Tests/MountebankSwiftTests/Api/Imposter+update.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+@testable import MountebankSwift
+
+extension Imposter {
+
+    func update(port: Int) -> Imposter {
```

Comment:
```suggestion
    func with(port: Int) -> Imposter {
```

Like this convention: 

https://developer.apple.com/documentation/musickit/musicpropertycontainer/with(_:)-10vhk/
https://developer.apple.com/documentation/uikit/uiimage/3327300-withtintcolor